modules/installation/pages/proxy-setup.adoc

```diff
@@ -229,9 +229,26 @@ If you do not use the reactivation key, you will need to re-register all the cli
 . In the {productname} {webui}, select the newly installed {productname} Proxy, and delete it from the systems list.
 . In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
   Click [guimenu]``Generate New Key``, and make a note of the new key.
-. Register the proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
-  In the bootstrap script set the reactivation key with the [systemitem]``ACTIVATION_KEYS`` parameter.
-  For more information about using reactivation keys, see xref:client-configuration:activation-keys.adoc[].
+
+For more information about using reactivation keys, see xref:client-configuration:activation-keys.adoc[].
+
+For Salt proxies, you need to do some additional steps before you bootstrap the new Proxy:
+
+
+
+.Procedure: Configuring Salt Proxy After Replacement
+. Ensure you have installed, reactivated, and re-reregistered the new proxy.
+. On the proxy, at the command prompt, check the machine ID, and make a note of the ID for later:
++
+----
+cat /etc/machine-id
+----
+. In the {webui}, navigate to menu:Salt[Keys], locate the Salt key associated with the proxy, and click btn:[delete].
```

Comment:
For salt proxy, this must be one of the first steps to do and not the last. Or what proxy are we talking here, new or old?

If you are replacing a proxy, there is a reasonable assumption that you will want to use old FQDN of the proxy.
Then you follow documentation - backup old proxy, shutdown old proxy, install new one with same FQDN and try to register it -> you are rejected by salt because of using existing minion-id, but different salt-key.

So the procedure for **salt based proxy** should be:
1) backup old proxy
2) shut it down
3) generate reactivation key for the proxy
4) remove salt key for old proxy (but only salt key, server entry must remain)
5) install new proxy
6) adapt bootstrap script and use reactivation key as activation key
7) register proxy
8) restore backuped if files if needed.

This needs to be confirmed with others however as this is what we eventually did with customer on call, but is a result of support session.

Reply:
OK, take another look.